misc/docs/source/tutorial/code_snippets/stationxml_custom_tags.rst

```diff
@@ -0,0 +1,167 @@
+.. _stationxml-extra:
+
+=============================================================================
+Handling custom defined tags in StationXML with the Obspy Inventory
+=============================================================================
+
+StationXML allows use of custom elements in addition to the 'usual' information
+defined by the StationXML standard. It allows *a)* custom namespace attributes to
+StationXML namespace tags and *b)* custom namespace subtags to StationXML namespace
+elements.
+ObsPy can handle both basic custom tags in the Inventory object's Network, Station, and Channel objects (*a*) and custom
+attributes (*b*) during input/output to/from StationXML.
+The following basic example illustrates how to output a StationXML file
+that contains additional xml tags/attributes:
+
+.. code-block:: python
+
+    from obspy import Inventory, UTCDateTime
+    from obspy.core.inventory import Network
+    from obspy.core.util import AttribDict
+
+    extra = AttribDict({'my_tag': {'value': True,
```

Comment:
Can you break the line before "'my_tag'"? Then it will look more consistent and I think the current version is not pep8 compatible.